static/app/components/acl/access.tsx

```diff
@@ -14,103 +15,75 @@ const DEFAULT_NO_ACCESS_MESSAGE = (
 );
 
 // Props that function children will get.
-export type ChildRenderProps = {
+type ChildRenderProps = {
   hasAccess: boolean;
   hasSuperuser: boolean;
 };
 
-type ChildFunction = (props: ChildRenderProps) => React.ReactNode;
+type ChildFunction = (props: ChildRenderProps) => JSX.Element;
 
-type DefaultProps = {
+type Props = {
   /**
    * List of required access levels
    */
-  access: Scope[];
-
+  access?: Scope[];
```

Comment:
I'm seeing a lot of the use cases for this are actually just a single access permission, do we want to support Scope | Scope[] here?

In react it makes a difference because every place where we pass <Component access={[scope]}/> we are creating a new array reference, probably not that big of a deal since it's a simple component, but it'd be nice to skip that if possible

Reply:
I think we're just going to try and kill this component off.

Reply:
But to do so I want to make sure everything can `useOrganization` safely